Setup32/Setup32.vdproj

```diff
@@ -347,7 +347,7 @@
         "DisplayName" = "8:Release"
         "IsDebugOnly" = "11:FALSE"
         "IsReleaseOnly" = "11:TRUE"
-        "OutputFilename" = "8:Release\\WasatchNET-Setup32.msi"
+        "OutputFilename" = "8:Release\\Setup32.msi"

```

Comment:
I kind of liked the old one better?

Reply:
I honestly don't know what this one is about lol, it might come out in the wash